src/gsIO/gsFileData.h

```diff
@@ -344,8 +344,11 @@ class gsFileData
             gsWarn <<"gsFileData: getAnyFirst: Didn't find any "<<
                 internal::gsXml<Object>::type()<<" "<< 
                 internal::gsXml<Object>::tag() <<". Error.\n";
-      }
-        internal::gsXml<Object>::get_into(node, result);// Using gsXmlUtils
+        }
```

Comment:
Or should we throw here a std::runtime_error through GISMO_ENSURE?

Reply:
No, that would cause massive changes because all the gsFileData functions only write to gsWarn and continue.

Reply:
I think it is fine as you did it..

Reply:
yes, this is what i tried to say.